components/log-viewer-webui/server/src/fastify-v2/plugins/app/search/QueryJobsDbManager/index.ts

```
@@ -0,0 +1,172 @@
+import type {MySQLPromisePool} from "@fastify/mysql";
+import {encode} from "@msgpack/msgpack";
+import settings from "@settings" with { type: "json" };
+import {FastifyInstance} from "fastify";
+import fp from "fastify-plugin";
+import {ResultSetHeader} from "mysql2";
+import {setTimeout} from "timers/promises";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)